index.html

```diff
@@ -10,12 +10,24 @@
         <header class="header">
             <div class="container">
                 <div class="header_body">
-                    <a class="header_logo">
+                    <a href="" class="header_logo">
                         <h1 id="renome">Renome</h1>
                     </a>
                     <div class="container_menu">
                         <div class="header_cart" id="header_cart">
-                            <button class="button_cart"><img src="/icons/cart.svg"/></button>
+                            <nav class="cart" id="cart"><img src="/icons/cart.svg"/>
+                                <ul class="cart_list" id="cart_list">
+                                    <li>
+                                        <a href="" class="cart_link">$44.50</a>
```

Comment:
Would be nice to have `#` inside the `href` as a placeholder, this way you will be able to test behavior of your links

Reply:
Ok, i did it